Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -1292,7 +1292,7 @@ checkCabalVersion pkg =
            && isNothing (setupBuildInfo pkg)
            && buildType pkg == Custom) $
       PackageBuildWarning $
-           "Packages using 'cabal-version: >= 1.24' with 'build-type: Custom' "
+           "Packages using 'cabal-version: 1.24' with 'build-type: Custom' "
```

Comment:
There is still a slight problem with this. It should be 1.24 _or higher_. `>=` conveyed that info, but it was syntactically incorrect.

```suggestion
           "Packages using 'cabal-version: 1.24' or higher with 'build-type: Custom' "
```

Reply:
Thanks, done!

Reply:
Oh, sorry, do you think, I should add "or higher" everywhere?

Reply:
Yes please!

Reply:
Done!